sources/osg/State.js

```diff
@@ -1321,6 +1313,28 @@ State.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( Objec
             this._checkCacheAndApplyUniform( uniform, cacheUniformsForeign, i, programUniformMap, name );
 
         }
+    },
+
+    // Use to detect changes in RenderLeaf between call to avoid to applyStateSet
+    _setStateSetsDrawID: function ( id ) {
+        var values = this.stateSets.values();
+        for ( var i = 0, nbStateSets = values.length; i < nbStateSets; i++ ) {
+            values[ i ].setDrawID( id );
+        }
+    },
+
+    _hasStateSetStackChangedFromPreviousDraw: function ( id, nbLast ) {
```

Comment:
what do you think @stephomi?  _hasStateSetStackChangedFromPreviousDraw/_stateSetStackChangedFromPreviousDraw/ or simply  _stateSetStackChanged